tests/v2/API/v3/Beneficiary/CreateTest.php

```diff
@@ -38,8 +47,16 @@ public function testCreateBeneficiary(): void
                 'email' => 'api@test.com',
                 'phone' => '1234567890',
                 'distant_id' => '1200',
-                '',
+                'external_center' => '42',
+                'external_pro_id' => '4972',
             ]
         );
+        $benef = $this->repo->findByUsername('api.test.13/02/2023');
+        $this->assertNotNull($benef);
```

Comment:
you could use Factories

Reply:
Can not find benef from username by using BeneficiaireFactory